docs/src/content/docs/resources/plugins.mdx

```diff
@@ -212,4 +212,9 @@ These community tools and integrations can be used to add features to your Starl
 		title="astro-plantuml"
 		description="Render PlantUML content in Markdown code blocks."
 	/>
+	<LinkCard
+		href="https://github.com/joesaby/astro-mermaid"
+		title="astro-mermaid"
+		description="Render Mermaid content in Markdown code blocks."
```

Comment:
```suggestion
		description="Client-side render Mermaid content in Markdown code blocks."
```

I think it may be important to clarify that the integration only supports client-side rendering of Mermaid diagrams. This may be a limitation for many documentation projects due to the trade-off of the approach so making it clear in the wording seems like a good idea.